apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/layout.tsx

```diff
@@ -52,7 +52,19 @@ export default async function ChainPageLayout({
   children: React.ReactNode;
   params: { chain_id: string };
 }) {
-  const chain = await getChain(params.chain_id);
+  // This is getting pulled from an api
+  // assumption is that it's getting pulled from a db
+  // Real use case: update db
+  // test use case: hard code
+  // const chain = await getChain(params.chain_id);
+  const chainResult = await getChain(params.chain_id);
+  const chain = {
+    ...chainResult,
+    services: [
+      ...chainResult.services,
+      { service: "faucet", enabled: chainResult.testnet },
```

Comment:
we should only add the "faucet" if its testnet, otherwise just... remove it - faucet is *always* there if testnet, it's *never* there if mainnet